workers/unity/Packages/io.improbable.gdk.core/.codegen/Source/Generators/Core/UnityComponentSenderGenerator.cs

```diff
@@ -31,7 +31,7 @@ public static CodeWriter Generate(UnityComponentDetails componentDetails)
                         partial.Type("internal class ComponentReplicator : IComponentReplicationHandler", replicator =>
                         {
                             replicator.Line($@"
-private ProfilerMarker componentMarker = new ProfilerMarker(""{componentDetails.Name}"");
+private ProfilerMarker componentMarker = new ProfilerMarker(""{componentDetails.Name}.SendUpdates"");
```

Comment:
This might require an internal changelog entry?

Reply:
This doesn't actually send updates, it queues them. Sending of the updates is done in a later system

Reply:
Added a couple more markers for sending + serialization